public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,239 @@
+openapi: 3.0.0
+info:
+  title: CTC Guarantee Balance
+  contact: {}
+  version: '2.0'
+servers:
+  - url: https://test-api.service.hmrc.gov.uk/customs/guarantees
+    description: Sandbox
+  - url: https://api.service.hmrc.gov.uk/customs/guarantees
+    description: Production
+    variables: {}
+paths:
+  /customs/guarantees/:grn/balance:
+    post:
+      tags:
+        - customs
+      summary: Get a Balance
+      description: Used to get the balance associated with the supplied Guarantee Reference Number (GRN)
+      operationId: GetaBalance
```

Comment:
Does this need to be qualified in some way? For v1.0, we fudged it by saying `SendaBalanceRequest`.

Suggestion:
getGuaranteeBalance

Reply:
Agree that this is a small API so value of `operationId` is less than usual.

For Traders API. `operationId` will be useful for, say, links: https://swagger.io/docs/specification/paths-and-operations/